frontend/src/components/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -27,6 +26,18 @@ type ReviewRow = {
 };
 
 const AdminDashboard = (): React.ReactElement => {
+  const [data, setData] = useState<ReviewResponse[]>([]);
+
+  useEffect(() => {
+    getReviews().then((allReviews: ReviewResponse[]) => {
+      const reviews: ReviewResponse[] = [];
```

Comment:
Is there a reason you couldn't just do `setData(allReviews)`?

Reply:
Sorry my bad, I was looking off of `Tags.tsx` which needed `forEach` but for `ReviewResponse` it's sufficient to just do `setData(allReviews)`. Will change!